src/runtime/pipeline/pipeline_executor.cc

```diff
@@ -79,8 +79,10 @@ PackedFunc PipelineExecutor::GetFunction(const std::string& name,
         [sptr_to_self, this](TVMArgs args, TVMRetValue* rv) { *rv = this->GetOutput(); });
   } else if (name == "run") {
     return PackedFunc([sptr_to_self, this](TVMArgs args, TVMRetValue* rv) { this->Run(args[0]); });
-  } else if (name == "stop") {
-    return PackedFunc([sptr_to_self, this](TVMArgs args, TVMRetValue* rv) { this->Stop(); });
+  } else if (name == "get_statistic_pipe_execute_number") {
+    return PackedFunc([sptr_to_self, this](TVMArgs args, TVMRetValue* rv) {
+      *rv = this->PipelineStatisticPipeExecuteNumber();
```

Comment:
As I said, it's not clear what `Number` means. How about `PipelineStatisticPipeExecuterCount()`

Reply:
fixed.